app/src/main/java/com/example/marvelapp/data/mapper/CharacterMapper.kt

```diff
@@ -26,14 +25,15 @@ object CharacterMapper {
             characterResponse.description,
             characterResponse.modified,
             transformThumbnail(characterResponse.thumbnail),
+            characterResponse.modified,
```

Comment:
why do you need this? this line is repeated in the line 26

Reply:
Thx, it was a typo mistake, instead of modified, the correct property was resourceUri

![image](https://user-images.githubusercontent.com/73836200/111639034-a58bfd00-87d9-11eb-9ee2-74089b54e91f.png)